docs/catalog/ai/leverage-sustainable-regions.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
+ - machine-learning
+ - role:data-scientist
+ - size:small
+---
+
+# Use sustainable regions for ML training 
+
+## Description
+Training an AI model has a significant carbon footprint. Leverage
+a region which offers carbon-free energy for running your ML training jobs.
+
+
+## Solution
+Use a cloud region which offers carbon-free energy for running your ML training workloads.
```

Comment:
Suggest changing the wording from "offers carbon-free energy" to "has a lower carbon intensity value". Energy may or may not be "carbon-free" and the pattern is to move to lower carbon emitting public cloud regions and shouldn't restrict to only carbon-free ones

Reply:
@dubrie , Done. Thanks